wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/hc/ServersChecker.java

```diff
@@ -56,16 +60,17 @@
    * @param machineName name of machine whose servers will be checked by this method
    * @param servers map of servers in a machine
    */
-  public ServersReadinessChecker(
+  @Inject
+  public ServersChecker(
       RuntimeIdentity runtimeIdentity,
       String machineName,
       Map<String, ? extends Server> servers,
-      ServerCheckerFactory serverCheckerFactory) {
+      @Assisted MachineTokenProvider machineTokenProvider) {
```

Comment:
Am I wrong that assisted annotation should be on other arguments, but not on this one?

Reply:
You're correct. I've already fixed it locally.